### ISSUE TITLE
iscsi deployment broken in pacific

### ISSUE THREAD

**Issue by smithfarm** [SUSE/sesdev#459]
Apparently, something got merged to `master` which broke the iSCSI deployment :-(

Reproducer:

```
sesdev --verbose create pacific
```

Symptom:

```
aster:~ # ceph orch ls --service-type iscsi
NAME                 RUNNING  REFRESHED  AGE  PLACEMENT    IMAGE NAME                                                               IMAGE ID   
iscsi.iscsi_service      0/2  3m ago     13m  node1;node2  registry.opensuse.org/filesystems/ceph/master/upstream/images/ceph/ceph  <unknown>  
```

... and it looks like there really are no iscsi daemons running.

**Comment by smithfarm** [comment]
@matthewoliver @ricardoasmarques @bk201 Is this something you'd be inclined to look into?

**Comment by matthewoliver** [comment]
I found it doesn't work on master at the moment, which I assume is also
Pacific. Problem is in ceph we've changed all blacklist -> blocklist. But
the same hasn't been done for ceph-iscsi. So it attempts to call 'ceph osd
blacklist rm' but that isn't a valid command anymore.

Ive already pushed a PR into ceph-iscsi to continue the blacklist ->
blocklist there which will fix it.

Ill double check this when I get to my computer (and link the PR).

Matt
